mitmproxy/tools/web/app.py

```diff
@@ -236,7 +236,16 @@ def wrapper(
             self: AuthRequestHandler, *args: P.args, **kwargs: P.kwargs
         ) -> R | None:
             if not self.current_user:
-                password = self.get_argument("token", default="")
+                password = ""
+                auth_header = self.request.headers.get("Authorization", None)
+                if auth_header:
+                    auth_parts = auth_header.split(" ", 1)
+                    if len(auth_parts) == 2 and auth_parts[0] == "Bearer":
+                        password = auth_parts[1]
```

Comment:
Code style nit:

```suggestion
                
                if auth_header := self.request.headers.get("Authorization"):
                    auth_scheme, _, auth_params = auth_header.partition(" ")
                    if auth_scheme == "Bearer":
                        password = auth_params
```

(named variables are easier to parse than list indices)

Reply:
Fixed with 7b04a12fda67d10b956cd886179aae3f377dd6fb